main.py

```diff
@@ -106,7 +103,8 @@ def del_memes(context):
         # lad_bot.edit_message_text(chat_id=meme_sent[2], message_id=meme_sent[1].message_id, text=edited_text)
 
     del MEMES_SENT[:]
```

Comment:
For better readability use-
```suggestion
    MEMES_SENT.clear()
```